### PR TITLE
Docs part 4.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/en/queries/create-command.md [hunk 1]
```diff
@@ -185,3 +185,18 @@ Yiisoft\Db\Query\Data\DataReader#4710
     )
 )
 ```
+
```
Comment (samdark) [inline docs/en/queries/create-command.md:188]: ????

**Review by samdark — APPROVED** [review]

File: docs/en/command.md [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$db->createCommand()->createTable('{{%customer}}', [
```
Comment (darkdef) [inline docs/en/command.md:17]: Need add string - better use db-migrate package, but you can create database with using columnSchemaBuilder

Comment (darkdef) [reply docs/en/command.md:17]: And need add example with string definitions for column

File: docs/en/command.md [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$db->createCommand()->createIndex('idx-customer-name', '{{%customer}}', 'name')->execute();
+```
```
Comment (darkdef) [inline docs/en/command.md:211]: May be add more examples for concrete dbms?

File: docs/en/command.md [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$db->createCommand()->dropUnique('uq-customer-name', '{{%customer}}')->execute();
+```
+
+## Batch insert
```
Comment (darkdef) [inline docs/en/command.md:407]: Split ddl and dml

**Review by darkdef — APPROVED** [review]

**Pull request merged**